CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Working on osm2streets
+
+Come hang out at https://github.com/a-b-street/osm2streets to ask questions and improve these docs.
+
+## Pre-requisites
+- rust beta
```

Comment:
rust nightly?

Reply:
I suggest just encoding this using https://github.com/a-b-street/osm2lanes/blob/main/rust-toolchain.toml

Reply:
rust beta seems to be working for me, if I understand my setup right, so I'd like to stay there if possible. I'll add the `rust-toolchain.toml`.